includes/uploadcare_settings.php

```diff
@@ -179,8 +147,43 @@
+
         <h3 id="uc-collapse-toggle" class="uc-show-hide uc-toggle"><?= __('Advanced options', 'uploadcare')?></h3>
         <div id="uc-advanced-options" data-toggle="uc-collapse-toggle" class="uc-collapsed hide">
+            <h4><?= __('Transfer WordPress Media Library to Uploadcare', 'uploadcare')?></h4>
+            <p><?= __('It\'ll move all previously uploaded files from your <code>/wp-content/uploads/</code> folder to Uploadcare cloud storage and then it\'ll update image URLs in all posts and pages automatically. The transfer is required if you want to use <a href="https://uploadcare.com/products/adaptive-delivery/" target="_blank">Adaptive Delivery</a> with Secure Uploading feature.', 'uploadcare')?></p>
+            <?php if (isset($_POST['uc_sync_data']) && $_POST['uc_sync_data'] === 'sync'): ?>
+                <div style="display: inline-block; border: solid 1px #23a100; padding: 0 10px; line-height: 30px; border-radius: 4px;">
+                    <strong><?= __('Transfer is in progress') ?></strong>
+                </div>
+            <?php else: ?>
+                <?php if ($loader->getHasLocalMedia() === false): ?>
+                    <div
+                        style="display: inline-block; border: solid 1px #23a100; padding: 0 10px; line-height: 30px; border-radius: 4px;">
+                        <strong style="color: #23A100"><?= __('All your Media Library files have been transfered successfully') ?></strong>
+                    </div>
+                <?php else: ?>
+                    <button <?= $connectError !== null ? 'disabled="disabled"' : null ?> class="button" type="submit" value="sync" name="uc_sync_data">
+                        <?php $loaderMediaCount = $loader->getLocalMediaCount() ?>
+                        <?= \sprintf(
+                            _n(
+                                'Transfer %d WordPress image to Uploadcare',
+                                'Transfer %d WordPress images to Uploadcare',
+                                $loaderMediaCount,
+                                'uploadcare'
+                            ),
+                            \number_format_i18n($loaderMediaCount)
+                        ) ?>
+                    </button>
+                <?php endif; ?>
+            <?php endif; ?>
+
+            <p><?= __("Your files are intact during the transfer. When the process is fully complete, the original WordPress files will be kept. We keep them because WordPress engine is not reliable and for now we can't ensure that all image URLs are replaced with a new ones.", 'uploadcare')?></p>
```

Comment:
```suggestion
            <p><?= __("Your files are intact during the transfer and after. We keep the original file duplicates because the WordPress engine is not reliable and some image URLs may mot be replaced with the new ones.", 'uploadcare')?></p>
```

Reply:
вот тут как происходит? мы храним дубликаты все время, да?

Reply:
В текущей версии да.